guessInput.js

```diff
@@ -16,7 +16,27 @@ const guessHandler = () => {
     }
 }
 
+let startGame = document.querySelector("#startGame")
 let guessInput = document.querySelector("#guessInput")
+let timerContainer = document.querySelector("#timerContainer")
+
+startGame.addEventListener("click", () => {
+    startGame.style.display = "none";
+    guessInput.style.display = "block";
+    let gameTimer = document.createElement('p')
+    gameTimer.innerText = 30
+    timerContainer.appendChild(gameTimer)
+    const interval = setInterval(() => {
+        gameTimer.innerText--
+        timerContainer.appendChild(gameTimer)
+        if (gameTimer.innerText == 0) {
```

Comment:
do we want to add in 'Time left:' or something similar next to the count down? Just a suggestion.